launcher/ui/instanceview/VisualGroup.cpp

```diff
@@ -66,6 +66,9 @@ void VisualGroup::update()
             rows[currentRow].height = maxRowHeight;
             rows[currentRow].top = offsetFromTop;
             currentRow++;
+            if(currentRow >= rows.size()) {
```

Comment:
```suggestion
            if (currentRow >= rows.size()) {
```

Reply:
Hey, I thought I did but ended up doing it wrongly, Will try to fix now.